.github/workflows/resync-build.yml

```diff
@@ -0,0 +1,123 @@
+# WARNING: This file is autogenerated - changes will be overwritten if not made via https://github.com/pulumi/ci-mgmt
```

Comment:
Curious what does this do, is it something working?

Reply:
It looks like it is a manually triggered repo action to sync with ci-mgmt. I didn't know it existed before, but it's apparently standard in our repos.